Tasks/Common/VstsAzureHelpers_/ImportFunctions.ps1

```diff
@@ -101,9 +101,12 @@ function Import-FromModulePath {
         }
 
         # Import the module.
+        $oldWarningPreference = $WarningPreference
+        $WarningPreference = "SilentlyContinue"
         Write-Host "##[command]Import-Module -Name $($module.Path) -Global"
-        $module = Import-Module -Name $module.Path -Global -PassThru -Force
+        $module = Import-Module -Name $module.Path -Global -PassThru -Force 3>$null
         Write-Verbose "Imported module version: $($module.Version)"
+        $WarningPreference = $oldWarningPreference
```

Comment:
https://github.com/microsoft/azure-pipelines-tasks/pull/12407 This PR also solving similar issue